utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -28,291 +29,248 @@
 import sys
 import docker
 import collections
+import random
+import string
 
-try:
-    from Atomic.mount import DockerMount
-    from Atomic.mount import MountError
-    import inspect
 
-    if "mnt_mkdir" not in inspect.getargspec(DockerMount.__init__).args:
-        sys.stderr.write(
-            "\"Atomic.mount.DockerMount\" has been successfully imported but "
-            "it doesn't support the mnt_mkdir argument. Please upgrade your "
-            "Atomic installation to 1.4 or higher.\n"
-        )
-        sys.exit(1)
-
-    # we only care about method names
-    member_methods = [
-        x[0] for x in
-        inspect.getmembers(
-            DockerMount, predicate=lambda member: inspect.isfunction(member) or inspect.ismethod(member)
-        )
-    ]
-
-    if "_clean_temp_container_by_path" not in member_methods:
-        sys.stderr.write(
-            "\"Atomic.mount.DockerMount\" has been successfully imported but "
-            "it doesn't have the _clean_temp_container_by_path method. Please "
-            "upgrade your Atomic installation to 1.4 or higher.\n"
-        )
-        sys.exit(1)
-
-except ImportError:
-    sys.stderr.write(
-        "Failed to import \"Atomic.mount.DockerMount\". It seems Atomic has "
-        "not been installed.\n"
-    )
-    sys.exit(1)
+def randomString(stringLength=10):
```

Comment:
Python standard library has the [UUID module](https://docs.python.org/2/library/uuid.html) exactly for this purpose.